compiler/testData/diagnostics/tests/Nullability.fir.kt

```diff
@@ -114,24 +114,24 @@ fun test() {
   }
   else {
     out?.println();
-    out.<!NONE_APPLICABLE!>println<!>();
+    out<!UNSAFE_CALL!>.<!>println();
   }
 
   if (out == null || out.println() == Unit) {
     out?.println();
-    out.<!NONE_APPLICABLE!>println<!>();
+    out<!UNSAFE_CALL!>.<!>println();
   }
   else {
     out.println();
   }
 
   if (1 == 2 || out != null && out.println(1) == Unit) {
     out?.println(2);
-    out.println(<!TOO_MANY_ARGUMENTS!>2<!>);
+    out.<!NONE_APPLICABLE!>println<!>(2);
```

Comment:
Any idea why we are getting `NONE_APPLICABLE` here instead of `UNSAFE_CALL`?

Reply:
It's because we are not mapping `ConeAmbiguityError` to unsafe calls. I added Commit#2 to do this.